src/App.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={-5} b={5} />
+    <Sum a={10} b={0} />
```

Comment:
You shouldn't pass `b` variable at all. It should looks like:
```suggestion
    <Sum a={10} />
```
![img1](https://user-images.githubusercontent.com/31217573/195061656-47459e64-26f5-4e98-8413-9b4b7dc437a5.jpg)